package.json

```diff
@@ -1,17 +1,25 @@
 {
   "name": "autobus",
-  "version": "0.1.6",
+  "version": "1.1.6",
```

Comment:
When bumping a higher value, the lower values are set to 0 again. Should be `1.0.0`.